### PR TITLE
executor: support Chunk for CheckTableExec

### PR DESCRIPTION
**Pull request by zz-jason**
to #5261

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: executor/executor.go [hunk 1]
```diff
@@ -262,39 +262,54 @@ type CheckTableExec struct {
...
 			}
 		}
 	}
-	e.done = true
-
-	return nil, nil
-}
-
-// Close implements plan.Plan Close interface.
-func (e *CheckTableExec) Close() error {
```
Comment (shenli) [inline executor/executor.go]: Why remove this function?

Comment (zz-jason) [reply executor/executor.go]: It's redundant, we can just use `baseExecutor.Close()`

**Comment by shenli** [comment]
LGTM

**Pull request merged**